### PR TITLE
{geo}[foss/2022a] pysteps v1.7.1 w/ Python 3.10.4

### PR DESCRIPTION
**Pull request by smoors**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by smoors** [comment]
@boegelbot: please test @ generoso

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in total)
node300.hydra.os - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz (skylake_avx512), Python 2.7.5
See https://gist.github.com/0e73838fc0ecc51b54a8d36c211befa2 for a full test report.

File: easybuild/easyconfigs/p/pysteps/pysteps-1.7.1-foss-2022a.eb [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        'patches': ['pysteps-1.7.1_h5py-3.7_compat.patch'],
+        'source_tmpl': 'v%(version)s.tar.gz',
+        'source_urls': ['https://github.com/pySTEPS/pysteps/archive/'],
+        'checksums': [
+            {'v1.7.1.tar.gz': '5063d7546d9995ec6ca52a5b4e50ef3f3c1ae43088a55e370af4aec18162a22f'},
+            {'pysteps-1.7.1_h5py-3.7_compat.patch': 'c2b1a47d7151a7f4072dce797c6303314c0d8fd7683b0001deb43f24357ed977'},
+        ],
+    }),
+    ('pysteps-nwp-importers', '6ca1582', {
+        'source_tmpl': '%(version)s.tar.gz',
```
Comment (boegel) [inline easybuild/easyconfigs/p/pysteps/pysteps-1.7.1-foss-2022a.eb:46]: @smoors Please rename on download, and use a better version (datestamp)

```suggestion
    ('pysteps-nwp-importers', '20220124', {
        'sources': [{'download_filename': '6ca1582.tar.gz', 'filename': SOURCE_TAR_GZ}],

File: easybuild/easyconfigs/p/pysteps/pysteps-1.7.1-foss-2022a.eb [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    ('jsmin', '3.0.1', {
+        'checksums': ['c0959a121ef94542e807a674142606f7e90214a2b3d1eb17300244bbb5cc2bfc'],
+    }),
+    ('pyFFTW', '0.13.0', {
+        'source_tmpl': '%(name)s-%(version)s.tar.gz',
+        'checksums': ['da85102405c0bd95d57eb19e99b01a0729d8406cb204c3900894b873784253da'],
+    }),
+    (name, version, {
+        'patches': ['pysteps-1.7.1_h5py-3.7_compat.patch'],
+        'source_tmpl': 'v%(version)s.tar.gz',
```
Comment (boegel) [inline easybuild/easyconfigs/p/pysteps/pysteps-1.7.1-foss-2022a.eb:38]: @smoors Please rename on download

```suggestion
        'sources': [{'download_filename': 'v%(version)s.tar.gz', 'filename': SOURCE_TAR_GZ}],
```

File: easybuild/easyconfigs/p/pysteps/pysteps-1.7.1-foss-2022a.eb [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+        'source_urls': ['https://github.com/pySTEPS/pysteps/archive/'],
+        'checksums': [
+            {'v1.7.1.tar.gz': '5063d7546d9995ec6ca52a5b4e50ef3f3c1ae43088a55e370af4aec18162a22f'},
```
Comment (boegel) [inline easybuild/easyconfigs/p/pysteps/pysteps-1.7.1-foss-2022a.eb:41]:
```suggestion
            {'pysteps-1.7.1.tar.gz': '5063d7546d9995ec6ca52a5b4e50ef3f3c1ae43088a55e370af4aec18162a22f'},

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by smoors** [comment]
@boegelbot: please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3103.skitty.os - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/5a58ff707ff31423614fd49c58fa030e for a full test report.

**Comment by boegel** [comment]
Going in, thanks @smoors!

**Pull request merged**